js/cadastroProduto.js

```diff
@@ -0,0 +1,18 @@
+       function CadastrarProduto() {
```

Comment:
No Javascript utilizamos a convenção [camelCase](https://www.dio.me/articles/camel-case-vs-pascal-case). Então o nome dos métodos e variáveis devem ficar conforme abaixo:
```suggestion
       function cadastrarProduto() {
```